<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QFrame" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#MainFrame {background-image: url(:/Images/resources/images/KPA_MainWindow_Dark_Skin.png);}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <widget class="QToasterDial" name="gainDial">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="noiseGateDial">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="modRateDial">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>16383.000000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="modIntensityDial">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="delayFeedbackDial">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>200.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>%</string>
   </property>
  </widget>
  <widget class="QToasterDial" name="reverbTimeDial">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QToasterDial" name="cleanSenseDial">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>18</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="minValue">
    <double>-12.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>12.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterDial" name="distortionSenseDial">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>84</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="minValue">
    <double>-12.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>12.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterButton" name="stompAButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>74</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="eqButton">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>64</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="stompBButton">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>64</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="stompCButton">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>64</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="stompDButton">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>74</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="stompXButton">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>74</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="stompModButton">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>64</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="rigUpButton">
   <property name="geometry">
    <rect>
     <x>817</x>
     <y>318</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="state">
    <enum>QToasterButton::Off</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="rigNextButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>356</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="state">
    <enum>QToasterButton::On</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="rigDownButton">
   <property name="geometry">
    <rect>
     <x>817</x>
     <y>394</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="state">
    <enum>QToasterButton::Off</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="rigPrevButton">
   <property name="geometry">
    <rect>
     <x>774</x>
     <y>356</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="state">
    <enum>QToasterButton::On</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="amplifierButton">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>74</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="cabinetButton">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>74</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="delayButton">
   <property name="geometry">
    <rect>
     <x>663</x>
     <y>64</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="reverbButton">
   <property name="geometry">
    <rect>
     <x>716</x>
     <y>74</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="stompsButton">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>118</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="stackButton">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>118</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="effectsButton">
   <property name="geometry">
    <rect>
     <x>633</x>
     <y>118</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QChickenHeadDial" name="chickenHeadDial">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>332</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="modeFrames">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>561</width>
     <height>331</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="TunerFrame" name="tuner"/>
   <widget class="BrowserFrame" name="browser"/>
   <widget class="QWidget" name="perform"/>
   <widget class="QWidget" name="profiler"/>
   <widget class="StompEditorFrame" name="stompEditor"/>
   <widget class="QWidget" name="ampEditor"/>
   <widget class="QWidget" name="eqEditor"/>
   <widget class="QWidget" name="cabEditor"/>
  </widget>
  <widget class="QToasterLookUpTableDial" name="rigVolumeDial">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>480</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterLookUpTableDial::Uni</enum>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterLookUpTableDial" name="headphoneVolumeDial">
   <property name="geometry">
    <rect>
     <x>803</x>
     <y>18</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterLookUpTableDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterLookUpTableDial::None</enum>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterLookUpTableDial" name="monitorVolumeDial">
   <property name="geometry">
    <rect>
     <x>761</x>
     <y>84</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterLookUpTableDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterLookUpTableDial::None</enum>
   </property>
   <property name="unit">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QToasterLookUpTableDial" name="reverbMixDial">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterLookUpTableDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterLookUpTableDial::None</enum>
   </property>
  </widget>
  <widget class="QToasterLookUpTableDial" name="delayMixDial">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterLookUpTableDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterLookUpTableDial::None</enum>
   </property>
  </widget>
  <widget class="QMultiColorLed" name="stompALed">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>30</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="color">
    <enum>QMultiColorLed::Off</enum>
   </property>
  </widget>
  <widget class="QMultiColorLed" name="stompBLed">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>20</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="color">
    <enum>QMultiColorLed::Off</enum>
   </property>
  </widget>
  <widget class="QMultiColorLed" name="stompCLed">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>20</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="color">
    <enum>QMultiColorLed::Off</enum>
   </property>
  </widget>
  <widget class="QMultiColorLed" name="stompDLed">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>30</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="color">
    <enum>QMultiColorLed::Off</enum>
   </property>
  </widget>
  <widget class="QMultiColorLed" name="stompModLed">
   <property name="geometry">
    <rect>
     <x>624</x>
     <y>20</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="color">
    <enum>QMultiColorLed::Off</enum>
   </property>
  </widget>
  <widget class="QMultiColorLed" name="stompXLed">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>30</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="color">
    <enum>QMultiColorLed::Off</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="inputButton">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>187</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
   <property name="state">
    <enum>QToasterButton::On</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="outputButton">
   <property name="geometry">
    <rect>
     <x>748</x>
     <y>187</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
   <property name="state">
    <enum>QToasterButton::On</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="storeButton">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>364</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>364</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QToasterButton" name="prevPageButton">
   <property name="geometry">
    <rect>
     <x>667</x>
     <y>422</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToasterButton" name="nextPageButton">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>422</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="state">
    <enum>QToasterButton::Off</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="masterVolumeWidget">
   <property name="geometry">
    <rect>
     <x>785</x>
     <y>150</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <widget class="QWidget" name="masterVolumeDBPage">
    <widget class="QToasterLookUpTableDial" name="masterVolumeDBDial">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterLookUpTableDial::None</enum>
     </property>
     <property name="unit">
      <string>dB</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="masterVolumePage">
    <widget class="QToasterDial" name="masterVolumeDial">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="knobSize">
      <enum>QToasterDial::Big</enum>
     </property>
     <property name="ledRingType" stdset="0">
      <enum>QToasterDial::None</enum>
     </property>
     <property name="maxValue">
      <double>10.000000000000000</double>
     </property>
     <property name="stepWidth">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToasterButton" name="tapButton">
   <property name="geometry">
    <rect>
     <x>845</x>
     <y>456</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="type">
    <enum>QToasterButton::Big</enum>
   </property>
  </widget>
  <widget class="QMultiColorLed" name="stompDelay2Led">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>20</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="color">
    <enum>QMultiColorLed::Off</enum>
   </property>
  </widget>
  <widget class="QMultiColorLed" name="stompDelay1Led">
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>20</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="color">
    <enum>QMultiColorLed::Off</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToasterDial</class>
   <extends>QWidget</extends>
   <header>qtoasterdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterLookUpTableDial</class>
   <extends>QWidget</extends>
   <header>QToasterLookUpTableDial.h</header>
  </customwidget>
  <customwidget>
   <class>QMultiColorLed</class>
   <extends>QWidget</extends>
   <header>qmulticolorled.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterButton</class>
   <extends>QWidget</extends>
   <header>qtoasterbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QChickenHeadDial</class>
   <extends>QWidget</extends>
   <header>qchickenheaddial.h</header>
  </customwidget>
  <customwidget>
   <class>BrowserFrame</class>
   <extends>QWidget</extends>
   <header>BrowserFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TunerFrame</class>
   <extends>QWidget</extends>
   <header>TunerFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StompEditorFrame</class>
   <extends>QWidget</extends>
   <header>StompEditorFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
